.env.example

```diff
@@ -80,6 +80,9 @@ PERIODICAL=stop_expired_deploys:60,remove_expired_locks:60,report_system_stats:6
 # NEW_RELIC_APP_NAME: Samson
 # NEW_RELIC_LOG_FILE_PATH: STDOUT
 
+## Datadog APM
+# ENABLE_DATADOG_APM=true
```

Comment:
would be nice to keep all related settings together when printing the ENV ... so DATADOG_APM_ENABLED=true